conf/prod.routes

```diff
@@ -8,6 +8,8 @@ GET         /customs-enrolment-services/language/:lang
 
 GET         /customs-enrolment-services/get-eori                                     @uk.gov.hmrc.customs.rosmfrontend.controllers.RegisterRedirectController.getEori()
 
+POST        /customs-enrolment-services/:serviceName/enrol                                   @uk.gov.hmrc.customs.rosmfrontend.controllers.EnrolWithCdsEnrolmentController.enrol(serviceName)
```

Comment:
I've gone for "service" as last part of url.   Don't mind which way we go but we need to be consistent if possible.

Reply:
Yeah, we will need to be consistent.

This is just a temporary thing, I think that this routes and controller will be removed and the service will be connected with your controller on POST method

Reply:
My controller doesn't currently do a post.  The "Continue" button can just be linked to your controller.